content/git/concepts/pull-requests/pull-requests.md

```diff
@@ -26,15 +26,35 @@ PRs are featured on popular version control platforms such as [GitHub](https://g
 
 ## Create a Pull Request
 
-The prerequisites for creating a PR and contributing to a project are:
+### Method 1: Fork (Ideal for Open Source Projects)
 
 1. Fork the project.
-2. Clone the forked repository to local machine.
-3. Create, and switch into, a new branch.
-4. Add, commit, and push the changes back to the repository.
+2. Clone the forked repository to your local machine.
+3. Create a new branch, then switch to it.
+4. `add`, `commit`, and `push` the changes to your forked repository.
+5. From the web page of your forked repository, click on the "Pull requests" tab.
 
-With the newly added changes pushed, open the web page of the repository. There should be a button that says "Create Pull Request" placed somewhere, similar to the image below.
+### Method 2: Collaborators (Ideal for Small Teams)
+If you are a collaborator of a repository (meaning the repository owner has added you as a collaborator), you can:
 
-![Repo page with Pull Request detect button](https://i.imgur.com/TsKCnyr.png)
+1. Clone the repository to your machine.
+2. Create a new branch, then switch to it.
+3. Make your changes, `commit`, and then `push` the branch to the repository.
+4. From the web page of the repository, click on the "Pull Requests" tab.
 
-Click that button and follow the steps to create the PR.
+### Method 3: Within an Organization/Company
+If you are part of an organization/company, depending on your permissions, you may have the privilege to push branches directly and create PRs (Pull Requests). You can follow the same steps as Method 2 to create a PR.
+
+> **Note:** Regardless of the method you choose, be sure to follow the contribution guidelines and standards specific to the project you are working on.
+
+No matter which method you use, you need to visit the relevant repository to access the PRs (Pull Requests).
+
+
+Navigate to the repository of the relevant project and go to 'Pull requests'.
```

Comment:
Please follow the instructions on file hosting in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting). You need to upload images to the `media\` folder. Do this for each of these image links. Please also give them names in lower case with dashes instead of spaces or underscores.